packages/trace-viewer/tsconfig.json

```
@@ -30,6 +30,6 @@
     },
     "useUnknownInCatchVariables": false,
   },
-  "include": ["src"],
+  "include": ["src", "../playwright/src/isomorphic/events.ts"],
```

Perhaps include all isomorphic files?

```suggestion
  "include": ["src", "../playwright/src/isomorphic"],
```